doctests/__init__.py

```diff
@@ -5,11 +5,31 @@
 import importlib
 import os
 import pkgutil
+import re
+import sys
+from doctest import OutputChecker
 
 import fuel
 from tests import skip_if_not_available
 
 
+class Py23DocChecker(OutputChecker):
+    """Single-source Python 2/3 output checker.
+
+    For more information, see the `original blog post`_.
+
+    .. _original blog post:
+       https://dirkjan.ochtman.nl/writing/2014/07/06/
+       single-source-python-23-doctests.html
+
+    """
+    def check_output(self, want, got, optionflags):
+        if sys.version_info[0] > 2:
+            want = re.sub("u'(.*?)'", "'\\1'", want)
+            want = re.sub('u"(.*?)"', '"\\1"', want)
+        return doctest.OutputChecker.check_output(self, want, got, optionflags)
```

Comment:
Why not `OutputChecker.check_output` (it's imported after all) or, even better, `super(Py23DocChecker, self).check_output(want, got, optionflags)`